CMakeLists.txt

```diff
@@ -43,7 +43,10 @@ endif()
 ################################################################################
 # project metadata
 ################################################################################
-project(HPX CXX C)
+if (HPX_WITH_CUDA)
```

Comment:
Is `HPX_WITH_CUDA` going to be set at this point already?

Reply:
yep because it is an option in the cmake command line no ?

Reply:
Oh, indeed, you're right. Thanks!